bats_test/get-mempool/get-mempool.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# Assertions
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+@test "'ardrive get-mempool | wc -l' outputs a number of less than 5 digits" {
```

Comment:
I'm not sure this test will stand the test of time, no pun intended 😬

Reply:
ahah its fine now, and an easy fix if we need to change it.